buildconfig/pygame-stubs/constants.pyi

```diff
@@ -0,0 +1,521 @@
+'''
```

Comment:
I guess there will be some constant differences depending on platform, python version, and SDL version.

Reply:
If that is the case than it could maybe cause some problems with mypy, because they don't want something like from pygame.constants import *; they want either pygame.constants to have type hints like this or to have __all__ (which would basically mean the same thing). Btw without this file it worked well in PyCharm (because is is able to see the constants file).